src/core/utils/humanize/smartSplit.ts

```diff
@@ -84,15 +90,22 @@ function* tokenize(source: string): Generator<SmartSplitToken> {
         switch (source[i]) {
             case ' ':
                 yield* yieldBlock();
-                yield { type: SmartSplitTokenType.BREAK, start: i, end: i + 1 };
+                yield { type: SmartSplitTokenType.BREAK, start: i, end: i + 1, content: ' ' };
                 break;
             case '"':
                 yield* yieldBlock();
-                yield { type: SmartSplitTokenType.QUOTE, start: i, end: i + 1 };
+                yield { type: SmartSplitTokenType.QUOTE, start: i, end: i + 1, content: '"' };
                 break;
             case '\\':
                 yield* yieldBlock();
-                yield { type: SmartSplitTokenType.ESCAPE, start: i, end: i + 1 };
+                yield {
+                    type: SmartSplitTokenType.LITERAL,
+                    start: i,
+                    end: ++i + 1,
```

Comment:
why do you double increment here?

Reply:
This token is 2 chars wide, the backslash and the following character.
Because this "consumes" the next character too i needs to be incremented so the next iteration doesnt consider it